packages/edit-site/src/components/global-styles/screen-revisions/style.scss

```diff
@@ -66,12 +75,10 @@
 		width: 100%;
 		height: auto;
 		display: block;
-		padding: $grid-unit-15 $grid-unit-15 $grid-unit-10 $grid-unit-30;
-		&:focus,
-		&:active {
-			outline: 0;
-			box-shadow: none;
-		}
+		padding: $grid-unit-15 $grid-unit-15 $grid-unit-10 $grid-unit-50;
+		z-index: 1;
+		position: relative;
```

Comment:
Without this style, the focus outline would be hidden from the timeline line.

![fcus-outline-overwrap](https://github.com/WordPress/gutenberg/assets/54422211/8b14c3c9-9786-4a35-b41a-acd945652e85)